M/MaximumIndependentSet/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+using BinaryBuilder
+
+name = "MaximumIndependentSet"
+version = v"0.1"
+sources = [
+    GitSource("https://github.com/claud10cv/MaximumIndependentSet.git", "3b37c113bd2e1b93bf38808f036632a731f0349b"),
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/MaximumIndependentSet
+make -j${nproc}
+make install
+"""
+
+platforms = supported_platforms()
+filter!(t -> os(t) in ["linux"], platforms)
+filter!(t -> arch(t) == "x86_64", platforms)
+filter!(t -> isnothing(libc(t)) ||  libc(t) == "glibc", platforms)
+
```

Comment:
```suggestion
platforms = expand_cxxstring_abis(platforms)

```
The auditor reports that `std::string` is used, so we need to handle the ABI differences.

Reply:
The tarballs now build for all platforms